commands/web/dkan-frontend-install

```diff
@@ -34,10 +29,6 @@ if [ ! -z "$DKAN_MODULE_PATH" ]; then
   fi
 fi
 
-drush pm-enable dkan_js_frontend -y
```

Comment:
Hey @janette, does the frontend work without the dkan_js_frontend module installed? If not, this seems like the right place to be enabling it -